packages/pangomm_1_4.rb

```diff
@@ -0,0 +1,44 @@
+require 'package'
+
+class Pangomm_1_4 < Package
```

Comment:
Wouldn't a better name be `Pangomm_2_46`?

Reply:
Same reason as w/ cairomm-1.0